logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -99,7 +99,7 @@ import java.util.Collections
 
 @Suppress("LongParameterList", "TooManyFunctions")
 class CallManagerImpl internal constructor(
-    private val calling: Calling,
+    private val calling: Calling?,
```

Comment:
Having a nullable `Calling` just because for some tests it is flaky looks very weird.